### PR TITLE
Added a file structure base resolving #5

### PR DESCRIPTION
**Pull request by blauertee**
We pretty much decided to use the file and directory structure as I proposed in #5. I created sample files so make clear what belongs where.
In our meeting we also decided that the modules folder is not necessary and we can put single file classes/modules in the `privacy-evaluator` folder.

### CODE REVIEW

**Review by budmil — APPROVED** [review]

**Review by Erdnaf — APPROVED** [review]

**Review by highclassfella — APPROVED** [review]

**Review by marisanest — APPROVED** [review]

**Comment by Erdnaf** [comment]
Hey @adrinjalali, 
We are wondering if we should paste the licence in every file, like we did in this PR or if it is better to do that only for the whole project in the root folder. 
What is your opinion bout that?

**Comment by adrinjalali** [comment]
You don't need to have the license notice on top of each file. The file on the root folder is definitely enough.

You sure you need an empty `__init__.py` file? I don't think it's needed in modern python. But you'd certainly need it once you actually have stuff in those folders.

**Comment by blauertee** [comment]
> You sure you need an empty __init__.py file?
I just created them so I could push the folders (since we'll need them anyway at some point), don't know weather it's possible to commit empty folders in git. If so I didn't figure that out yet.

File: tests/sample_test.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# MIT License
```
Comment (adrinjalali) [inline tests/sample_test.py:1]: this file still has the license.

**Comment by marisanest** [comment]
> I just created them so I could push the folders (since we'll need them anyway at some point), don't know weather it's possible to commit empty folders in git. If so I didn't figure that out yet.

You can just add a .gitignore file into the folders. Then you can commit these folders to git without them having any other content.

**Review by jtorhoff — APPROVED** [review]

**Comment by adrinjalali** [comment]
> You can just add a .gitignore file into the folders. Then you can commit these folders to git without having any content.

That's also an alternative, but these folders probably need an `__init__.py` file anyway, so I guess it's okay.

**Review by Friedrich-Mueller — APPROVED** [review]

**Review by Friedrich-Mueller — APPROVED** [review]

**Pull request merged**